### ISSUE TITLE
Bottom column bar displayed under every subtable after piping table to `group-by --to-table`

### ISSUE THREAD

**Issue by Yethal** [nushell/nushell#14060]
### Describe the bug

When a table is tall enough that the column bar does not fit on screen nushell displays the column bar at the bottom of the table. However after piping the table to `group-by --to-table` bottom bar is displayed under the subtables instead of the top-level table. Moreover it is displayed only under those subtables which on their own are tall enough not to fit on screen

### How to reproduce

```
#!/usr/bin/env nu
seq 1 200
| each {touch $"($in).txt"}

ls
| group-by type --to-table
```

![image](https://github.com/user-attachments/assets/82c39769-5523-44f2-8902-e566625e722c)

### Expected behavior

I expected nu to display bottom bar under the outer table

### Configuration

| key                | value                                                             |
| ------------------ | ----------------------------------------------------------------- |
| version            | 0.98.0                                                            |
| major              | 0                                                                 |
| minor              | 98                                                                |
| patch              | 0                                                                 |
| branch             |                                                                   |
| commit_hash        |                                                                   |
| build_os           | linux-x86_64                                                      |
| build_target       | x86_64-unknown-linux-gnu                                          |
| rust_version       | rustc 1.80.1 (3f5fd8dd4 2024-08-06) (built from a source tarball) |
| cargo_version      | cargo 1.80.0 (376290515 2024-07-16)                               |
| build_time         | 1980-01-01 00:00:00 +00:00                                        |
| build_rust_channel | release                                                           |
| allocator          | mimalloc                                                          |
| features           | default, sqlite, trash                                            |
| installed_plugins  | formats 0.98.0, gstat 0.98.0, polars 0.98.0, query 0.98.0         |

**Comment by fdncred** [comment]
I'm not following. I just ran your script, and it has both the header and the footer, not only the footer.

At the top
![image](https://github.com/user-attachments/assets/2e0393ca-5edf-44bc-ae80-271d095cce59)
At the bottom
![image](https://github.com/user-attachments/assets/8f5aa1e9-e102-4f38-80a4-3dbcd814ded4)

Maybe you're saying that `group` and `items` should appear at the bottom?

**Comment by Yethal** [comment]
Yes, that's what I meant. The footer bar should appear at the bottom of the outer table

**Comment by Yethal** [comment]
The problem with displaying it under subtables and not under the main table is that tables with many subtables will occupy more vertical space in order to display essentially duplicated information

**Comment by fdncred** [comment]
I bet it's kind of complicated to change if we want sub tables to respect the config setting. The main table should, to be sure.

**Comment by zhiburt** [comment]
Interesting proposition @Yethal

Yes it was never worked like it as I understand.

I think it's doable.
We could just indicate to outer table whether a threshold was reached in the inner one.
And if so the outer one gets to render it's footer.

Allthough I guess it's shouldn't be a default?

Did a patchwork not well tested though.

![image](https://github.com/user-attachments/assets/c42db07a-a826-4080-9475-ee709d0a1d5f)

**Comment by zhiburt** [comment]
And I'd like to point out that maybe we shall track the inner tables rows?
What I mean is if 2 tables aren't big enough for footer but there combination is far enough for a outer table to use a footer.
I think you get the point.

What I did is just simple if there's some footer inside - use it outside.

**Comment by Yethal** [comment]
Also, I do believe it's valuable to have footers under every subtable if the subtables contain different columns (as a result of transposing a complex record?), it's only in this case where it should be avoided

**Comment by fdncred** [comment]
reopened because of this https://github.com/nushell/nushell/pull/14070#issuecomment-2433048475

**Comment by fdncred** [comment]
@zhiburt any thoughts on this? https://github.com/nushell/nushell/pull/14070#issuecomment-2433048475

**Comment by NotTheDr01ds** [comment]
> And I'd like to point out that maybe we shall track the inner tables rows?

Just getting around to testing this out, but I think that's the real answer to this problem.  This solution (nor `footer_mode: auto`) still doesn't handle things like `scope modules`, where there are fewer rows in both the inner and outer tables than the terminal height.

**Comment by zhiburt** [comment]
> where there are fewer rows in both the inner and outer tables than the terminal height.

And according to your @NotTheDr01ds  example,
a combination of tables (rows) has more then enough to render footer right?
In which case we want to render footer in outer one correct?

So if you wish I could try to do that.
To account for the **"whole"** table size.

______

And sorry @fdncred wasn't able to check the issue yet.

**Comment by NotTheDr01ds** [comment]
> In which case we want to render footer in outer one correct?
> 
> So shall I try to do that? To account for the **"whole"** table size.

I don't have strong feelings on it, but I think do it's a nice "polish" thing, and it certainly would extend your current PR to handle more cases.

**Comment by fdncred** [comment]
no worries @zhiburt. please look at it when you get time.
